allowable/src/lib.rs

```diff
@@ -64,10 +64,12 @@ impl<'a> Allowable<'a> {
     }
 
     pub fn is_allowed(&self, deps: &Deps, addr: Addr) -> StdResult<bool> {
-        if !(self.enabled.load(deps.storage)?) {
-            Ok(true)
-        } else {
-            Ok(self.allowed_addrs.has(deps.storage, addr))
+        match self.enabled.may_load(deps.storage).unwrap() {
+            Some(true) => Ok(self.allowed_addrs.has(deps.storage, addr)),
```

Comment:
Here we gracefully handle the case where the allowable module isn't instantiated and hence